pom.xml

```diff
@@ -237,6 +237,15 @@
     <error-prone.version>2.24.0</error-prone.version>
     <error-prone-support.version>0.14.0</error-prone-support.version>
     <doxia.version>1.12.0</doxia.version>
+    <error-prone.configuration-args>
+      <!-- Disabled rules as we do not see value in them. -->
+      -Xep:JUnitClassModifiers:OFF
```

Comment:
```suggestion
      <!-- until https://github.com/checkstyle/checkstyle/issues/8252 -->
      -Xep:JUnitClassModifiers:OFF
```

Reply:
We need to close this issue, we are not going to follow this style for now, may be in future we will, but I don't see much reason.

Reply:
Issue is closed.
Please make comment `<!-- reason at https://github.com/checkstyle/checkstyle/issues/8252 -->`

Reply:
>We need to close this issue, we are not going to follow this style for now, may be in future we will, but I don't see much reason.

Reasoning is sound: we do not want to allow/encourage sharing of test methods across test classes.

Reply:
Changing visibility does not protect again reuse, we are open for any reuse of public method, usually it is nonsense and not possible to have any reason.